RainbowVis.js

```diff
@@ -29,7 +29,7 @@ class ColorGradient {
         }
     };
 
-    colorAt(number) {
+    colourAt(number) {
```

Comment:
Ok for that but can you add a colorAt that just calls colourAt ? (or the opposite)